resources/lib/vrtplayer/tokenresolver.py

```diff
@@ -75,7 +75,8 @@ def get_cookie_from_cookiejar(cookiename, cookiejar):
 
     def _get_new_playertoken(self, path, token_url, headers):
         import json
-        req = Request(token_url, data='', headers=headers)
+        req = Request(token_url, headers=headers)
+        req.get_method = lambda: 'POST'
```

Comment:
No offence, but that looks... strange. Why the lambda function here ?

Reply:
Because `get_method` is a function and a string is not callable.